README.md

```diff
@@ -72,6 +72,8 @@ The bootstrap task currently installs the agent via the Puppet Enterprise packag
 
 For an extensive list of supported operating systems, see [metadata.json](https://github.com/puppetlabs/puppetlabs-bootstrap/blob/master/metadata.json)
 
+The bootstrap task is tested to work with Windows Management Framework >= v3.0+ and .NET >= v4.6 on the supported Windows operating systems.
```

Comment:
I think that it should be `.NET >= 4.5` here per https://blogs.perficient.com/2016/04/28/tsl-1-2-and-net-support/